src/util/auth-header.js

```diff
@@ -0,0 +1,10 @@
+export function authHeader() {
+    // return authorization header with jwt token
+    let user = JSON.parse(localStorage.getItem('user'));
```

Comment:
@nikhilrayaprolu 
can use `const` rather than `let` for this kind of variables.